### ISSUE TITLE
Can't build nvidia-x11 on unstable

### ISSUE THREAD

**Issue by lovesegfault** [NixOS/nixpkgs#74523]
**Describe the bug**
My system build is failing because I can't build nvidia-x11

**To Reproduce**
1. `nix-build '<nixos>' -A linuxPackages_latest.nvidia_x11`

```
these derivations will be built:
  /nix/store/303yil6ivmhrrjklsirblcppa58wxidw-nvidia-x11-435.21-5.4.drv
building '/nix/store/303yil6ivmhrrjklsirblcppa58wxidw-nvidia-x11-435.21-5.4.drv'...
unpacking sources
tail: error writing 'standard output': Broken pipe
Creating directory NVIDIA-Linux-x86_64-435.21
Verifying archive integrity... OK
Uncompressing NVIDIA Accelerated Graphics Driver for Linux-x86_64 435.21/nix/store/rbkl5y5bi9kbgcj26pxqbdp9nx3jmnjw-NVIDIA-Linux-x86_64-435.21.run: line 699: /build/makeself.NawASaTi/xz: No such file or directory
tail: error writing 'standard output': Broken pipe
/nix/store/rbkl5y5bi9kbgcj26pxqbdp9nx3jmnjw-NVIDIA-Linux-x86_64-435.21.run: line 690: /dev/tty: No such device or address

/nix/store/ya7i4fq09q27mkf0jc5g9n6mld6q9wdn-builder.sh: line 10:     5 Terminated              sh $src -x
./10_nvidia.json
./LICENSE
./nvidia-cuda-mps-server
./nvidia-application-profiles-435.21-key-documentation
./libGLESv2.so.2.1.0
./nvidia_drv.so
./README.txt
./mkprecompiled
./libglvnd_install_checker/
./libglvnd_install_checker/check-libglx
./libglvnd_install_checker/egl_installertest.json
./libglvnd_install_checker/check-libegl
./libglvnd_install_checker/check-entrypoint
./libglvnd_install_checker/check-libglvnd-install.sh
./libglvnd_install_checker/check-libgldispatch
./libglvnd_install_checker/libGLX_installertest.so.0
./libglvnd_install_checker/libEGL_installertest.so.0
./libnvidia-glsi.so.435.21
./libnvidia-compiler.so.435.21
./html/
./html/supportedchips.html
./html/i2c.html
./html/nvidia-smi.html
./html/commonproblems.html
./html/depth30.html
./html/dynamicpowermanagement.html
./html/newusertips.html
./html/egpu.html
./html/xrandrextension.html
./html/knownissues.html
./html/nvidia-ml.html
./html/installedcomponents.html
./html/glxsupport.html
./html/appendices.html
./html/xcompositeextension.html
./html/nvidiasettings.html
./html/addtlresources.html
./html/introduction.html
./html/installationandconfiguration.html
./html/sdi.html
./html/displaydevicenames.html
./html/optimus.html
./html/index.html
./html/gpunames.html
./html/framelock.html
./html/retpoline.html
./html/configmultxscreens.html
./html/xconfigoptions.html
./html/profiles.html
./html/primerenderoffload.html
./html/vdpausupport.html
./html/editxconfig.html
./html/nvidia-persistenced.html
./html/dma_issues.html
./html/xineramaglx.html
./html/acknowledgements.html
./html/nvidia-debugdump.html
./html/configtwinview.html
./html/configlaptop.html
./html/programmingmodes.html
./html/minimumrequirements.html
./html/audiosupport.html
./html/openglenvvariables.html
./html/procinterface.html
./html/addressingcapabilities.html
./html/selectdriver.html
./html/installdriver.html
./html/faq.html
./html/powermanagement.html
./html/kms.html
./html/sli.html
./html/randr14.html
./html/flippingubb.html
./html/dpi.html
./nvidia-debugdump
./libGLX.so.0
./nvidia-suspend.service
./nvidia-cuda-mps-control
./nvidia-resume.service
./libGLESv2_nvidia.so.435.21
./nvidia.icd
./libnvidia-cbl.so.435.21
./libnvidia-ptxjitcompiler.so.435.21
./libnvidia-ifr.so.435.21
./nvidia-bug-report.sh
./libnvidia-encode.so.435.21
./libnvidia-opencl.so.435.21
./nvidia-xconfig
./libGL.so.1.7.0
./nvidia-settings.1.gz
./nvidia-installer
./libnvidia-gtk2.so.435.21
./nvidia-installer.1.gz
./nvidia-modprobe
./libnvidia-tls.so.435.21
./pkg-history.txt
./libnvidia-fatbinaryloader.so.435.21
./makeself-help-script.sh
./nvidia-smi.1.gz
./nvidia-smi
./libnvidia-glcore.so.435.21
./libnvidia-rtcore.so.435.21
./libnvidia-fbc.so.435.21
./libGLESv1_CM.so.1.2.0
./nvidia_icd.json
./10_nvidia_wayland.json
./libOpenGL.so.0
./libnvidia-eglcore.so.435.21
./makeself.sh
./nvidia-modprobe.1.gz
./libnvidia-egl-wayland.so.1.1.3
./libGLESv1_CM_nvidia.so.435.21
./libGLX_nvidia.so.435.21
./libEGL.so.1.1.0
./nvidia-persistenced.1.gz
./nvidia-sleep.sh
./libEGL_nvidia.so.435.21
./libcuda.so.435.21
./libnvidia-cfg.so.435.21
./libvdpau_nvidia.so.435.21
./.manifest
./32/
./32/libGLESv2.so.2.1.0
./32/libglvnd_install_checker/
./32/libglvnd_install_checker/check-libglx
./32/libglvnd_install_checker/egl_installertest.json
./32/libglvnd_install_checker/check-libegl
./32/libglvnd_install_checker/check-entrypoint
./32/libglvnd_install_checker/check-libglvnd-install.sh
./32/libglvnd_install_checker/check-libgldispatch
./32/libglvnd_install_checker/libGLX_installertest.so.0
./32/libglvnd_install_checker/libEGL_installertest.so.0
./32/libnvidia-glsi.so.435.21
./32/libnvidia-compiler.so.435.21
./32/libGLX.so.0
./32/libGLESv2_nvidia.so.435.21
./32/libnvidia-ptxjitcompiler.so.435.21
./32/libnvidia-ifr.so.435.21
./32/libnvidia-encode.so.435.21
./32/libnvidia-opencl.so.435.21
./32/libGL.so.1.7.0
./32/libnvidia-tls.so.435.21
./32/libnvidia-fatbinaryloader.so.435.21
./32/libnvidia-glcore.so.435.21
./32/libnvidia-fbc.so.435.21
./32/libGLESv1_CM.so.1.2.0
./32/libOpenGL.so.0
./32/libnvidia-eglcore.so.435.21
./32/libGLESv1_CM_nvidia.so.435.21
./32/libGLX_nvidia.so.435.21
./32/libEGL.so.1.1.0
./32/libEGL_nvidia.so.435.21
./32/libcuda.so.435.21
./32/libvdpau_nvidia.so.435.21
./32/libGLdispatch.so.0
./32/libnvidia-glvkspirv.so.435.21
./32/libGL.la
./32/libnvcuvid.so.435.21
./32/libOpenCL.so.1.0.0
./32/libnvidia-ml.so.435.21
./32/libnvidia-opticalflow.so.435.21
./libGLdispatch.so.0
./libnvidia-gtk3.so.435.21
./libnvidia-glvkspirv.so.435.21
./nvidia-settings.png
./nvidia_layers.json
./libGL.la
./nvidia-xconfig.1.gz
./kernel/
./kernel/Makefile
./kernel/Kbuild
./kernel/nvidia-uvm/
./kernel/nvidia-uvm/uvm8_gpu.h
./kernel/nvidia-uvm/uvm8_procfs.h
./kernel/nvidia-uvm/uvm8_pascal_fault_buffer.h
./kernel/nvidia-uvm/uvm8_policy.c
./kernel/nvidia-uvm/uvm8_va_space.c
./kernel/nvidia-uvm/uvm8_perf_utils.h
./kernel/nvidia-uvm/uvm8_maxwell.c
./kernel/nvidia-uvm/uvm8_hmm.h
./kernel/nvidia-uvm/uvm8_processors.h
./kernel/nvidia-uvm/uvm8_lock.h
./kernel/nvidia-uvm/uvm8_va_space.h
./kernel/nvidia-uvm/clc46f.h
./kernel/nvidia-uvm/uvm_common.c
./kernel/nvidia-uvm/uvm8_volta_mmu.c
./kernel/nvidia-uvm/uvm8_procfs.c
./kernel/nvidia-uvm/clc3b5.h
./kernel/nvidia-uvm/uvm8_kepler.c
./kernel/nvidia-uvm/uvm8_hal.c
./kernel/nvidia-uvm/uvm8_mmu.c
./kernel/nvidia-uvm/uvm8_api.h
./kernel/nvidia-uvm/uvm8_global.c
./kernel/nvidia-uvm/uvm8_volta_access_counter_buffer.c
./kernel/nvidia-uvm/uvm8_map_external.c
./kernel/nvidia-uvm/uvm8_mmu_test.c
./kernel/nvidia-uvm/uvm8_gpu_semaphore.c
./kernel/nvidia-uvm/uvm8_ce_test.c
./kernel/nvidia-uvm/hwref/
./kernel/nvidia-uvm/hwref/turing/
./kernel/nvidia-uvm/hwref/turing/tu102/
./kernel/nvidia-uvm/hwref/turing/tu102/dev_fault.h
./kernel/nvidia-uvm/hwref/turing/tu102/dev_mmu.h
./kernel/nvidia-uvm/hwref/volta/
./kernel/nvidia-uvm/hwref/volta/gv100/
./kernel/nvidia-uvm/hwref/volta/gv100/dev_fault.h
./kernel/nvidia-uvm/hwref/volta/gv100/dev_mmu.h
./kernel/nvidia-uvm/hwref/volta/gv100/dev_fb.h
./kernel/nvidia-uvm/hwref/kepler/
./kernel/nvidia-uvm/hwref/kepler/gk104/
./kernel/nvidia-uvm/hwref/kepler/gk104/dev_mmu.h
./kernel/nvidia-uvm/hwref/pascal/
./kernel/nvidia-uvm/hwref/pascal/gp100/
./kernel/nvidia-uvm/hwref/pascal/gp100/dev_fault.h
./kernel/nvidia-uvm/hwref/pascal/gp100/dev_mmu.h
./kernel/nvidia-uvm/hwref/pascal/gp100/dev_fb.h
./kernel/nvidia-uvm/uvm_linux.c
./kernel/nvidia-uvm/uvm8_pascal_host.c
./kernel/nvidia-uvm/cla06fsubch.h
./kernel/nvidia-uvm/uvm8_maxwell_host.c
./kernel/nvidia-uvm/uvm8_turing_fault_buffer.h
./kernel/nvidia-uvm/nvstatus.c
./kernel/nvidia-uvm/uvm8_channel_test.c
./kernel/nvidia-uvm/uvm8_perf_heuristics.h
./kernel/nvidia-uvm/uvm8_ats_faults.c
./kernel/nvidia-uvm/uvm8_push_macros.h
./kernel/nvidia-uvm/cla16f.h
./kernel/nvidia-uvm/uvm_common.h
./kernel/nvidia-uvm/uvm8_perf_module_test.c
./kernel/nvidia-uvm/uvm8_pmm_gpu.c
./kernel/nvidia-uvm/uvm8_range_allocator.c
./kernel/nvidia-uvm/uvm8_pmm_test.c
./kernel/nvidia-uvm/uvm8_pmm_sysmem_test.c
./kernel/nvidia-uvm/uvm8_gpu_non_replayable_faults.c
./kernel/nvidia-uvm/uvm8_populate_pageable.c
./kernel/nvidia-uvm/clc06f.h
./kernel/nvidia-uvm/cla06f.h
./kernel/nvidia-uvm/uvm8_extern_decl.h
./kernel/nvidia-uvm/uvm8_tracker.c
./kernel/nvidia-uvm/uvm8_user_channel.h
./kernel/nvidia-uvm/uvm8_range_tree.h
./kernel/nvidia-uvm/uvm8_gpu_access_counters.h
./kernel/nvidia-uvm/uvm8_perf_events.c
./kernel/nvidia-uvm/uvm8_va_range.c
./kernel/nvidia-uvm/uvm8_channel.c
./kernel/nvidia-uvm/uvm8_page_tree_test.c
./kernel/nvidia-uvm/uvm8_range_tree.c
./kernel/nvidia-uvm/uvm8_push.c
./kernel/nvidia-uvm/uvm8_peer_identity_mappings_test.c
./kernel/nvidia-uvm/uvm8_mmu.h
./kernel/nvidia-uvm/uvm8_populate_pageable.h
./kernel/nvidia-uvm/cla0b5.h
./kernel/nvidia-uvm/uvm8_tracker.h
./kernel/nvidia-uvm/uvm8_range_group_tree_test.c
./kernel/nvidia-uvm/clc36f.h
./kernel/nvidia-uvm/uvm8_va_range.h
./kernel/nvidia-uvm/uvm8_rm_mem_test.c
./kernel/nvidia-uvm/uvm8_map_external.h
./kernel/nvidia-uvm/uvm8_perf_prefetch.c
./kernel/nvidia-uvm/uvm_unit_test.h
./kernel/nvidia-uvm/uvm8_pushbuffer.c
./kernel/nvidia-uvm/uvm8_gpu_semaphore_test.c
./kernel/nvidia-uvm/uvm8_hal.h
./kernel/nvidia-uvm/uvm_ioctl.h
./kernel/nvidia-uvm/nv-kthread-q-selftest.c
./kernel/nvidia-uvm/uvm8_mem_test.c
./kernel/nvidia-uvm/uvm8_lock.c
./kernel/nvidia-uvm/uvm8_range_allocator_test.c
./kernel/nvidia-uvm/ctrl2080mc.h
./kernel/nvidia-uvm/uvm_utils.h
./kernel/nvidia-uvm/uvm8_perf_heuristics.c
./kernel/nvidia-uvm/uvm8_kepler_mmu.c
./kernel/nvidia-uvm/uvm8_ats_ibm.c
./kernel/nvidia-uvm/uvm8_kepler_host.c
./kernel/nvidia-uvm/uvm8_range_group.c
./kernel/nvidia-uvm/uvm8_gpu_replayable_faults.c
./kernel/nvidia-uvm/uvm8_pte_batch.c
./kernel/nvidia-uvm/uvm8_tools.h
./kernel/nvidia-uvm/uvm8_kvmalloc_test.c
./kernel/nvidia-uvm/uvm8_gpu_non_replayable_faults.h
./kernel/nvidia-uvm/uvm8_user_channel.c
./kernel/nvidia-uvm/uvm8_test.h
./kernel/nvidia-uvm/uvm8_kepler_ce.c
./kernel/nvidia-uvm/uvm8_gpu_access_counters.c
./kernel/nvidia-uvm/uvm8_perf_thrashing.h
./kernel/nvidia-uvm/uvm8_migrate.h
./kernel/nvidia-uvm/uvm8_migrate_pageable.c
./kernel/nvidia-uvm/uvm8_push_test.c
./kernel/nvidia-uvm/uvm8_pmm_gpu.h
./kernel/nvidia-uvm/uvm8_pascal_ce.c
./kernel/nvidia-uvm/uvm8_test.c
./kernel/nvidia-uvm/uvm8_get_rm_ptes_test.c
./kernel/nvidia-uvm/uvm8_ats_faults.h
./kernel/nvidia-uvm/uvm8_range_allocator.h
./kernel/nvidia-uvm/uvm8_gpu_replayable_faults.h
./kernel/nvidia-uvm/uvm8_va_block_types.h
./kernel/nvidia-uvm/uvm8_perf_events.h
./kernel/nvidia-uvm/uvm_debug_optimized.c
./kernel/nvidia-uvm/uvm_utils.c
./kernel/nvidia-uvm/uvm8_volta_host.c
./kernel/nvidia-uvm/uvm8_range_group.h
./kernel/nvidia-uvm/uvm8_kvmalloc.h
./kernel/nvidia-uvm/uvm_linux.h
./kernel/nvidia-uvm/uvm8_ats_ibm.h
./kernel/nvidia-uvm/uvm8_tracker_test.c
./kernel/nvidia-uvm/uvm8_pushbuffer.h
./kernel/nvidia-uvm/uvm8.c
./kernel/nvidia-uvm/uvm8_rm_mem.c
./kernel/nvidia-uvm/uvm8_gpu.c
./kernel/nvidia-uvm/uvm8_range_tree_test.c
./kernel/nvidia-uvm/uvm8_migrate_pageable.h
./kernel/nvidia-uvm/uvm8_gpu_semaphore.h
./kernel/nvidia-uvm/uvm8_migrate.c
./kernel/nvidia-uvm/uvm8_pte_batch.h
./kernel/nvidia-uvm/clb06f.h
./kernel/nvidia-uvm/uvm8_pmm_sysmem.c
./kernel/nvidia-uvm/uvm8_thread_context.h
./kernel/nvidia-uvm/clc365.h
./kernel/nvidia-uvm/uvm8_thread_context.c
./kernel/nvidia-uvm/clc5b5.h
./kernel/nvidia-uvm/nvidia-uvm.Kbuild
./kernel/nvidia-uvm/uvm8_global.h
./kernel/nvidia-uvm/clc076.h
./kernel/nvidia-uvm/uvm8_va_block_test.c
./kernel/nvidia-uvm/uvm8_perf_module.c
./kernel/nvidia-uvm/uvm8_turing_mmu.c
./kernel/nvidia-uvm/uvm8_perf_utils_test.c
./kernel/nvidia-uvm/uvm8_lock_test.c
./kernel/nvidia-uvm/uvm8_volta.c
./kernel/nvidia-uvm/nvCpuUuid.c
./kernel/nvidia-uvm/uvm8_va_space_mm.h
./kernel/nvidia-uvm/uvm8_turing.c
./kernel/nvidia-uvm/uvm8_volta_fault_buffer.c
./kernel/nvidia-uvm/uvm8_hal_types.h
./kernel/nvidia-uvm/uvm8_perf_thrashing.c
./kernel/nvidia-uvm/uvm_linux_ioctl.h
./kernel/nvidia-uvm/uvm8_fault_buffer_flush_test.c
./kernel/nvidia-uvm/uvm8_turing_host.c
./kernel/nvidia-uvm/uvm8_forward_decl.h
./kernel/nvidia-uvm/uvm8_volta_fault_buffer.h
./kernel/nvidia-uvm/uvm8_va_space_mm.c
./kernel/nvidia-uvm/clb0b5.h
./kernel/nvidia-uvm/uvm8_test_rng.c
./kernel/nvidia-uvm/uvm8_pascal_fault_buffer.c
./kernel/nvidia-uvm/uvm8_va_block.h
./kernel/nvidia-uvm/uvm8_channel.h
./kernel/nvidia-uvm/uvm8_perf_utils.c
./kernel/nvidia-uvm/uvm8_perf_prefetch.h
./kernel/nvidia-uvm/uvm8_hmm.c
./kernel/nvidia-uvm/nvidia-uvm-sources.Kbuild
./kernel/nvidia-uvm/clb069.h
./kernel/nvidia-uvm/uvm8_tlb_batch.c
./kernel/nvidia-uvm/nv-kthread-q.c
./kernel/nvidia-uvm/uvm8_va_block.c
./kernel/nvidia-uvm/uvm8_test_ioctl.h
./kernel/nvidia-uvm/uvm8_gpu_isr.h
./kernel/nvidia-uvm/clc369.h
./kernel/nvidia-uvm/uvm8_rm_mem.h
./kernel/nvidia-uvm/clc0b5.h
./kernel/nvidia-uvm/uvm8_tlb_batch.h
./kernel/nvidia-uvm/uvm8_pascal_mmu.c
./kernel/nvidia-uvm/uvm8_tools_init.h
./kernel/nvidia-uvm/uvm8_thread_context_test.c
./kernel/nvidia-uvm/uvm8_tools.c
./kernel/nvidia-uvm/uvm8_test_rng.h
./kernel/nvidia-uvm/uvm8_perf_events_test.c
./kernel/nvidia-uvm/uvm8_pmm_sysmem.h
./kernel/nvidia-uvm/uvm8_mem.h
./kernel/nvidia-uvm/uvm8_push.h
./kernel/nvidia-uvm/clc1b5.h
./kernel/nvidia-uvm/uvm8_mem.c
./kernel/nvidia-uvm/uvm8_pascal.c
./kernel/nvidia-uvm/uvm.h
./kernel/nvidia-uvm/uvm8_gpu_isr.c
./kernel/nvidia-uvm/uvm8_kvmalloc.c
./kernel/nvidia-uvm/uvm8_perf_module.h
./kernel/nvidia-drm/
./kernel/nvidia-drm/nvidia-drm-gem.h
./kernel/nvidia-drm/nvidia-drm-os-interface.h
./kernel/nvidia-drm/nvidia-drm.Kbuild
./kernel/nvidia-drm/nvidia-drm-connector.c
./kernel/nvidia-drm/nvidia-drm-crtc.h
./kernel/nvidia-drm/nvidia-drm-gem-user-memory.c
./kernel/nvidia-drm/nvidia-drm-fb.h
./kernel/nvidia-drm/nvidia-drm-encoder.h
./kernel/nvidia-drm/nvidia-drm-drv.h
./kernel/nvidia-drm/nvidia-drm-conftest.h
./kernel/nvidia-drm/nvidia-drm-drv.c
./kernel/nvidia-drm/nvidia-drm-modeset.c
./kernel/nvidia-drm/nv-pci-table.c
./kernel/nvidia-drm/nvidia-drm-connector.h
./kernel/nvidia-drm/nvidia-drm-encoder.c
./kernel/nvidia-drm/nvidia-drm-crtc.c
./kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.c
./kernel/nvidia-drm/nvidia-drm-fb.c
./kernel/nvidia-drm/nvidia-drm-prime-fence.c
./kernel/nvidia-drm/nvidia-drm-priv.h
./kernel/nvidia-drm/nvidia-drm-linux.c
./kernel/nvidia-drm/nvidia-dma-fence-helper.h
./kernel/nvidia-drm/nvidia-drm-helper.h
./kernel/nvidia-drm/nvidia-drm-utils.c
./kernel/nvidia-drm/nvidia-drm-utils.h
./kernel/nvidia-drm/nv-pci-table.h
./kernel/nvidia-drm/nvidia-drm-helper.c
./kernel/nvidia-drm/nvidia-drm-modeset.h
./kernel/nvidia-drm/nvidia-drm.c
./kernel/nvidia-drm/nvidia-drm-ioctl.h
./kernel/nvidia-drm/nvidia-drm-gem.c
./kernel/nvidia-drm/nvidia-drm-gem-user-memory.h
./kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.h
./kernel/nvidia-drm/nvidia-drm.h
./kernel/nvidia-drm/nvidia-drm-prime-fence.h
./kernel/nvidia-modeset/
./kernel/nvidia-modeset/nv-modeset-kernel.o_binary
./kernel/nvidia-modeset/nvidia-modeset-os-interface.h
./kernel/nvidia-modeset/nvkms.h
./kernel/nvidia-modeset/nvidia-modeset-linux.c
./kernel/nvidia-modeset/nvkms-ioctl.h
./kernel/nvidia-modeset/nv-kthread-q.c
./kernel/nvidia-modeset/nvidia-modeset.Kbuild
./kernel/conftest.sh
./kernel/dkms.conf
./kernel/nvidia/
./kernel/nvidia/nv-cray.c
./kernel/nvidia/nv-report-err.c
./kernel/nvidia/nvlink_pci.h
./kernel/nvidia/rmil.h
./kernel/nvidia/nvlink_export.h
./kernel/nvidia/nv-pat.h
./kernel/nvidia/nvlink_linux.h
./kernel/nvidia/nv-kernel.o_binary
./kernel/nvidia/os-interface.c
./kernel/nvidia/nvlink_linux.c
./kernel/nvidia/ioctl_nvswitch.h
./kernel/nvidia/nv-procfs.c
./kernel/nvidia/nvlink_common.h
./kernel/nvidia/nv.c
./kernel/nvidia/os-pci.c
./kernel/nvidia/nv-gvi.c
./kernel/nvidia/rmp2pdefines.h
./kernel/nvidia/export_nvswitch.h
./kernel/nvidia/nv-pci.c
./kernel/nvidia/nv-p2p.h
./kernel/nvidia/nv-report-err.h
./kernel/nvidia/nv-vm.c
./kernel/nvidia/nv-i2c.c
./kernel/nvidia/nv-modeset-interface.c
./kernel/nvidia/os-registry.c
./kernel/nvidia/nv-pci-table.c
./kernel/nvidia/nv-p2p.c
./kernel/nvidia/nv_gpu_ops.h
./kernel/nvidia/nv-reg.h
./kernel/nvidia/nvlink_errors.h
./kernel/nvidia/nv-vtophys.c
./kernel/nvidia/nv-ibmnpu.c
./kernel/nvidia/linux_nvswitch.c
./kernel/nvidia/nv-pat.c
./kernel/nvidia/nv-mmap.c
./kernel/nvidia/os-mlock.c
./kernel/nvidia/os-usermap.c
./kernel/nvidia/nv-dma.c
./kernel/nvidia/nv-frontend.h
./kernel/nvidia/nv-msi.c
./kernel/nvidia/nv-frontend.c
./kernel/nvidia/nv-ibmnpu.h
./kernel/nvidia/nvlink_proto.h
./kernel/nvidia/nvidia-sources.Kbuild
./kernel/nvidia/nv-mempool.c
./kernel/nvidia/nv-pci-table.h
./kernel/nvidia/nv-rsync.h
./kernel/nvidia/nv_uvm_interface.c
./kernel/nvidia/nvidia.Kbuild
./kernel/nvidia/nv-acpi.c
./kernel/nvidia/nv-kthread-q.c
./kernel/nvidia/nv-usermap.c
./kernel/nvidia/nv-memdbg.c
./kernel/nvidia/nv-rsync.c
./kernel/common/
./kernel/common/inc/
./kernel/common/inc/nv.h
./kernel/common/inc/nv-kernel-interface-api.h
./kernel/common/inc/nvkms-kapi.h
./kernel/common/inc/uvmtypes.h
./kernel/common/inc/nv_uvm_interface.h
./kernel/common/inc/nv-register-module.h
./kernel/common/inc/nvgputypes.h
./kernel/common/inc/nv-gpu-info.h
./kernel/common/inc/conftest.h
./kernel/common/inc/nvlimits.h
./kernel/common/inc/nvmisc.h
./kernel/common/inc/nv-procfs.h
./kernel/common/inc/nv-pgprot.h
./kernel/common/inc/nv-msi.h
./kernel/common/inc/nv_speculation_barrier.h
./kernel/common/inc/os-interface.h
./kernel/common/inc/cpuopsys.h
./kernel/common/inc/nv-memdbg.h
./kernel/common/inc/nvstatuscodes.h
./kernel/common/inc/nv_uvm_types.h
./kernel/common/inc/nv-retpoline.h
./kernel/common/inc/nv-hypervisor.h
./kernel/common/inc/nv-timer.h
./kernel/common/inc/nvipmi.h
./kernel/common/inc/nvkms-api-types.h
./kernel/common/inc/nv-lock.h
./kernel/common/inc/nv-linux.h
./kernel/common/inc/nv-proto.h
./kernel/common/inc/nv-misc.h
./kernel/common/inc/nv-modeset-interface.h
./kernel/common/inc/nv-time.h
./kernel/common/inc/nv-mm.h
./kernel/common/inc/nvstatus.h
./kernel/common/inc/nvCpuUuid.h
./kernel/common/inc/nv-kthread-q.h
./kernel/common/inc/nv-kref.h
./kernel/common/inc/nv-list-helpers.h
./kernel/common/inc/nv-pci.h
./kernel/common/inc/nvtypes.h
./kernel/common/inc/xapi-sdk.h
./libnvcuvid.so.435.21
./nvidia-settings.desktop
./nvidia-drm-outputclass.conf
./nvidia-cuda-mps-control.1.gz
./nvidia
./nvidia-persistenced-init.tar.bz2
./nvidia-hibernate.service
./libnvoptix.so.435.21
./libOpenCL.so.1.0.0
./nvidia-persistenced
./libEGL.so.435.21
./nvidia-application-profiles-435.21-rc
./libnvidia-ml.so.435.21
./nvidia-settings
./NVIDIA_Changelog
./libnvidia-opticalflow.so.435.21
./libglxserver_nvidia.so.435.21
source root is .
setting SOURCE_DATE_EPOCH to timestamp 1566742269 of file ./nvidia-bug-report.sh
patching sources
configuring
no configure script, doing nothing
building
Building linux driver against kernel: /nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev
make[1]: Entering directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source'
make[2]: Entering directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/build'
The Module.symvers file is missing, or does not contain any
symbols exported from the kernel. This could cause the NVIDIA
kernel modules to be built against a configuration that does
not accurately reflect the actual target kernel.
The Module.symvers file check can be disabled by setting the
environment variable IGNORE_MISSING_MODULE_SYMVERS to 1.
make[3]: *** [/build/kernel/Kbuild:181: module_symvers_sanity_check] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source/Makefile:1652: /build/kernel] Error 2
make[2]: Leaving directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/build'
make[1]: *** [Makefile:179: sub-make] Error 2
make[1]: Leaving directory '/nix/store/lbl993a8wdv9q2zai1jkcaqscwgjc47y-linux-5.4-dev/lib/modules/5.4.0/source'
make: *** [Makefile:81: modules] Error 2
builder for '/nix/store/303yil6ivmhrrjklsirblcppa58wxidw-nvidia-x11-435.21-5.4.drv' failed with exit code 2
error: build of '/nix/store/303yil6ivmhrrjklsirblcppa58wxidw-nvidia-x11-435.21-5.4.drv' failed
```

**Expected behavior**
I was expecting it to build normally.

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.11, NixOS, 20.03pre202088.e89b21504f3 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre203172.0ee0489d42e"`
 - channels(bemeurer): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

@andir, @gilligan: I have no clue who maintains `nvidia-x11`, there seems to be no one listed at the package's `default.nix`

**Comment by veprbl** [comment]
cc @baracoder @eadwu

**Comment by baracoder** [comment]
Hm, it looks like 435.21 does not support linux 5.4.
The release notes of 440.31 have some notes of fixing this error https://www.nvidia.com/Download/driverResults.aspx/153226/

> Updated the Module.symvers sanity check, which is part of the NVIDIA kernel module build process, to accommodate the recent addition of a new field in the Module.symvers file format. This fixes the error "The Module.symvers file is missing [...]" seen during driver installation or DKMS rebuilds with Linux 5.4 RC kernels.
Fixed kernel module build problems with Linux kernel 5.4.0 release candidates.

If you can stay on linux 5.3 for now, you can try

```
nix-build '<nixos>' -A linuxPackages_5_3.nvidia_x11
```

**Comment by lovesegfault** [comment]
@baracoder Ah, this makes sense,I've disabled my Nvidia stuff for now and will be waiting on #73167

**Comment by veprbl** [comment]
Should be fixed by #73167

**Comment by lovesegfault** [comment]
@eadwu @baracoder @veprbl Y'all rock!
